horton/scripts/wpart.py

```diff
@@ -161,7 +164,8 @@ def wpart_slow_analysis(wpart, mol):
     if dm_spin is None:
         if not hasattr(mol, 'exp_alpha'):
             return
-        xs_response = 2*compute_noninteracting_response(mol.exp_alpha, operators)
+        xs_response = 2 * compute_noninteracting_response(mol.exp_alpha,
+                                                          operators)
```

Comment:
Just a minor note: We've set the maximum line length to 100. It is a good idea to set your editor a little more conservativelye, e.g. 90, but we found 80 a little too harsh.